Makefile

```
@@ -1,4 +1,5 @@
-.PHONY: build run test test-cover test-e2e lint ci clean docker-build docker-run
+.PHONY: build run test test-cover test-e2e lint ci clean docker-build docker-run \
+       vm-ssh vm-logs vm-restart vm-stop vm-start vm-status vm-deploy
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)